test/test_login_registration.py

```diff
@@ -171,18 +174,16 @@ def test_blank_username_login_submission_response(self):
         login_form.__setitem__("password", "12345678")
         response = login_form.submit()
 
-        # checks if HTTP response code is 303 (= See Other)
-        assert_equal(response.status, 303)
-
-        redirected = response.follow()
-        assert_equal(redirected.status, 200)
+        # checks if Validating page loaded
+        response.mustcontain(self.VALIDATING_TITLE)
+        assert_equal(response.status, 200)
 
```

Comment:
why is here a `assert_equal(response.status, 200)` instead of the `assert_equal(response.status, 303)` we had previously?